File: ContactManagerWeb/HarrisContactWeb/HarrisContactWeb/Models/BusinessContact.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace HarrisContactWeb.Models
{
    public class BusinessContact
    {//Attributes of BusinessContact Class
        public int BusinessContactID { get; set; }//Primary key for Business Contacts
        public string BusinessFname { get; set; }//String Business Contact Firstname
        public string BusinessLname { get; set; }//String Business Contact Lastname 
        public string BusinessEmail { get; set; }//String Business Contact Email
        public string BusinessTel { get; set; }//String Business Contact Telephone
        public string BusinessAdrr1 { get; set; }//String Business Contact Address 1
        public string BusinessAddr2 { get; set; }//String Business Contact Address 2
        public string BusinessCity { get; set; }//String Business Contact City
        public string BuisnessPostCode { get; set; }//String Business Contact Postcode

    }
}

File: ContactManagerJava/HarrisContactManager/src/BusinessContact.java
//Business contact inherits from Contact
public class BusinessContact extends Contact {
//Encapsulation of BusinessTel attribute
	private String BusniessTel;
//inherited attributes from contact class
	public BusinessContact(String id, String firstName, String lastName, String email, String addr1, String addr2,
			String city, String postCode, String busniessTel) {
		//attributes of super class 
		super(id, firstName, lastName, email, addr1, addr2, city, postCode);
		BusniessTel = busniessTel;
	}
//get method for business tel
	public String getBusniessTel() {
		return BusniessTel;
	}
//set method for business tel
	public void setBusniessTel(String busniessTel) {
		BusniessTel = busniessTel;
	}

	// Polymorphism-overriding toString method inherited from the contact class to String 
	@Override
	public String toString() {
		return "BusinessContact [BusniessTel=" + BusniessTel + "]";
	}

}
